src/Makefile

```diff
@@ -186,7 +188,7 @@ PACKMOST = \
 	ml-iap \
 	phonon
 
-# NOTE: the last three packages must remain at the end since
+# NOTE ^^^^^: the last three packages must remain at the end since
```

Comment:
@akohlmey typo?

Reply:
No, this is intentional. Comments are usually before relevant sections, but this is after because the list is so long. So I thought a little visual hint might be helpful.